include/pistache/listener.h

```diff
@@ -66,6 +70,11 @@ class Listener {
 
     void pinWorker(size_t worker, const CpuSet& set);
 
+#ifdef PISTACHE_USE_SSL
+    void setupSSL(std::string cert_path, std::string key_path, bool use_compression);
```

Comment:
IMO from the performance perspective it's may be better to pass both, cert_path and key_path as 
constant references (const std::string&) rather than copies of the strings (std::string)